src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -176,6 +176,34 @@ const useClusterPolling = namespace => {
   return { clusters, refreshClusters }
 }
 
+const useAppPolling = namespace => {
+  const signal = Utils.useCancellation()
+  const timeout = useRef()
+  const [apps, setApps] = useState()
+  const reschedule = ms => {
+    clearTimeout(timeout.current)
+    timeout.current = setTimeout(refreshAppsSilently, ms)
+  }
+  const loadApps = async () => {
+    try {
+      const newApps = await Ajax(signal).Apps.list(namespace, { creator: getUser().email })
+      setApps(newApps)
+      const app = currentApp(newApps)
+      reschedule(_.includes((app && (app.patchInProgress ? 'LeoReconfiguring' : app.status)), ['PROVISIONING', 'RUNNING']) ? 10000 : 120000)
```

Comment:
Is `patchInProgress` used on apps? Also, I don't think we should use the short timeout for the `RUNNING` status.